### PR TITLE
Small but important: free plans aren't billed

### PR DESCRIPTION
**Pull request by alshakero**
#### Changes proposed in this Pull Request

Remove "per month, billed yearly" from the free plan. And use "Free forever" instead.

### CODE REVIEW

**Comment by michelleweber** [comment]
Copy looks fine.

**Comment by yansern** [comment]
Maybe not showing anything would be more ideal? 100&cid=CNMU1UHHB-slack-CNMU1UHHB

image

File: client/landing/gutenboarding/components/plans/plans-table/plan-item.tsx [hunk 1]
```diff
@@ -67,7 +69,9 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 					<div className="plan-item__price-amount" data-is-loading={ ! price }>
 						{ price || nbsp }
 					</div>
-					<div className="plan-item__price-note">{ __( 'per month, billed yearly' ) }</div>
+					<div className="plan-item__price-note">
+						{ isFree ? __( 'free forever' ) : __( 'per month, billed yearly' ) }
```
Comment (simison) [inline client/landing/gutenboarding/components/plans/plans-table/plan-item.tsx:73]: Alternatively, we could check for the price to be `0` (regardless of currency)

Comment (alshakero) [reply client/landing/gutenboarding/components/plans/plans-table/plan-item.tsx:73]: The price has a currency symbol that might come before or after it.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/overview/3561347

Hi @alshakero, could you pleaes edit the description of this PR and add a screenshot for our translators. Ideally it'd include all the following strings:
- `free forever`

Thank you in advance!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/overview/3561347

Hi @alshakero, could you pleaes edit the description of this PR and add a screenshot for our translators. Ideally it'd include this string: `free forever`

Thank you in advance!

**Comment by alshakero** [comment]
@akirk @a8ci18n done.

**Comment by akirk** [comment]
Great, thanks! We updated the deliverable automatically.

**Pull request merged**